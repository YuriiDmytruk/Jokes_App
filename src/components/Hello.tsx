import * as React from 'react';

const Hello = (): JSX.Element => <div>Hello</div>;

export default Hello;
