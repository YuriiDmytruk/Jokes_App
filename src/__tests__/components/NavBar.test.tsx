import React from 'react';
import { render } from '@testing-library/react';
import Hello from '../../components/Hello'; // Import your component

// This will automatically generate and update snapshots
test('renders NavBar correctly', () => {
  const { asFragment } = render(<Hello />);
  expect(asFragment()).toMatchSnapshot();
});
