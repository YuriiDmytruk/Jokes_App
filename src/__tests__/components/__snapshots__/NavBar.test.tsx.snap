// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NavBar correctly 1`] = `
<DocumentFragment>
  <nav
    class="bg-body-tertiary navbar navbar-expand-lg navbar-light"
  >
    <div
      class="container"
    >
      <a
        href="/"
      >
        <span
          class="navbar-brand"
        >
          Jokes
        </span>
      </a>
    </div>
  </nav>
</DocumentFragment>
`;
